win/reindex.bat

```diff
@@ -0,0 +1 @@
+hushd.exe -reindex
```

Comment:
It's sad that we need this :(

Reply:
Until we find what cause all those people having to reindex, this little shortcut will save a lot of time explaining how to reindex on Discord.